### ISSUE TITLE
hb:static tasks runs long with many partials

### ISSUE THREAD

**Issue by alxbenz** [biotope/biotope-build#31]
When using many partials the hb:static tasks gets slower. Currently running around 10 seconds in on of the projects.

This needs to be way faster.

We probably need to find a way to not always rebuild all partials and maybe only update the ones that are changed. Currently I have no idea how to fix this...

**Comment by janrembold** [comment]
I complete re-wrote the static:hb task, see https://github.com/biotope/biotope-build/tree/hb-static-performance

This is not extensively tested by now, but the first execution times are really promising. For me it changed the execution time from 5s to 350ms :)

Please test this in your projects by adding "https://github.com/biotope/biotope-build.git#hb-static-performance" to your package.json.

You might need to change your local handlebars.helper.js from 
```
module.exports.register = factory;
```
to 
```
module.exports = factory;
```

**Comment by alxbenz** [comment]
- reworked the indexr to use the new hb2:static task (this needs testing on mac)
- removed old indexr and hb:static task
- updated index.hbs in biotope to match the new indexr task (https://github.com/biotope/biotope/tree/build_5.3.0_update)
- default bioHelpers get injected into the frontend, so they can be used during runtime

What needs to be done:
- rework browser matrix task to use new hb2:static task
- test indexr on mac (create new page, modify yaml attributes, delete page)

... ?

**Comment by timomayer** [comment]
@alxbenz thanks a lot please also update CHANGELOG.md
since this is a massive change please relrease 5.3.0-rc1 first, test it in one of our bigger projects, then release 5.3.0 :)

**Comment by janrembold** [comment]
@alxbenz  I just updated biotope-build branch and the corresponding biotope branch. 
- Browser Matrix is now integrated in hb2 task
- Indexr works fine on mac

I also added a MIGRATE.md file to root folder and copied existing migration checklists into this file. We need to add all 5.3.x migration steps there during RC testing.

I still see an Uglify error during `yarn build`. Haven't found the bug til now. Therefore I didn't release the RC1 by now
